src/Aspire.ProjectTemplates/templates/aspire-servicedefaults/8.2/Extensions.cs

```diff
@@ -91,8 +91,7 @@ private static IHostApplicationBuilder AddOpenTelemetryExporters(this IHostAppli
     public static IHostApplicationBuilder AddDefaultHealthChecks(this IHostApplicationBuilder builder)
     {
         builder.Services.AddHealthChecks()
-            // Add a default liveness check to ensure app is responsive
-            .AddCheck("self", () => HealthCheckResult.Healthy(), ["live"]);
+            .AddApplicationStatus();
```

Comment:
You can revert changes to the 8.2 templates. We don't support 8.x anymore so they're only there for convenience.